File: app/src/main/java/com/lookuptalk/model/LoginData.java

package com.lookuptalk.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class LoginData {

    @SerializedName("success")
    @Expose
    private Boolean success;
    @SerializedName("errorCode")
    @Expose
    private Integer errorCode;
    @SerializedName("description")
    @Expose
    private String description;
    @SerializedName("info")
    @Expose
    private Info info;

    /**
     * No args constructor for use in serialization
     * 
     */
    public LoginData() {
    }

    /**
     * 
     * @param success
     * @param errorCode
     * @param description
     * @param info
     */
    public LoginData(Boolean success, Integer errorCode, String description, Info info) {
        super();
        this.success = success;
        this.errorCode = errorCode;
        this.description = description;
        this.info = info;
    }

    public Boolean getSuccess() {
        return success;
    }

    public void setSuccess(Boolean success) {
        this.success = success;
    }

    public Integer getErrorCode() {
        return errorCode;
    }

    public void setErrorCode(Integer errorCode) {
        this.errorCode = errorCode;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public Info getInfo() {
        return info;
    }

    public void setInfo(Info info) {
        this.info = info;
    }

}

File: app/src/main/java/com/lookuptalk/model/LoginData.kt
package com.lookuptalk

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class LoginResponse {
    @SerializedName("success")
    @Expose
    var success: Boolean? = null

    @SerializedName("errorCode")
    @Expose
    var errorCode: Int? = null

    @SerializedName("description")
    @Expose
    var description: String? = null

    @SerializedName("info")
    @Expose
    var info: Any? = null

    /**
     * No args constructor for use in serialization
     *
     */
    constructor() {}

    /**
     *
     * @param success
     * @param errorCode
     * @param description
     * @param info
     */
    constructor(
        success: Boolean?,
        errorCode: Int?,
        description: String?,
        info: Any?
    ) : super() {
        this.success = success
        this.errorCode = errorCode
        this.description = description
        this.info = info
    }

}